### PR TITLE
Count lines report on travis

### PR DESCRIPTION
**Pull request by vncoelho**

### CODE REVIEW

**Comment by vncoelho** [comment]
Thanks @lock9, work in partnership here in São Paulo! :dagger:

File: .travis.yml [hunk 1]
```diff
@@ -11,14 +11,18 @@ mono: none
...
   - cd neo.UnitTests
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then ulimit -n 2048; fi
 
 script:
  - dotnet restore
  - find * -name *.csproj | xargs -I % dotnet add % package coverlet.msbuild
  - dotnet test /p:CollectCoverage=true /p:CoverletOutputFormat=opencover
-
+ - echo "Reporting number of lines with cloc"
+ - cloc ..
```
Comment (lock9) [inline .travis.yml]: Where is this info going?

Comment (vncoelho) [reply .travis.yml]: only travis for now, a bot could report it, but I think that on travis is enough

Comment (shargon) [reply .travis.yml]: We need it?

**Comment by erikzhang** [comment]
What's this?

**Comment by lock9** [comment]
@vncoelho  maybe we should use the badge only.
@erikzhang  it prints this in travis console:
Screen Shot 2019-07-08 at 23 42 13

**Comment by lock9** [comment]
@vncoelho  why is XML being included as code?

**Comment by shargon** [comment]
I think that we don't need it.

**Comment by erikzhang** [comment]
I'm not sure if this is useful.

**Comment by vncoelho** [comment]
It is useful, it does not consume much time and give us a nice summary.
Those looking at the travis will surely like. EHuahueahuea

I think that these kind of information on Travis are good and open doors for other analyses.
But fell free to close, @erikzhang, @shargon,@igormcoelho and @lock9.

**Pull request closed without merging**